source/declarators.tex

```diff
@@ -2105,16 +2105,16 @@
 
 \pnum
 In the \grammarterm{function-body}, a
-\grammarterm{function-local predefined variable} denotes a block-scope object of static
+\techterm{function-local predefined variable} denotes a block-scope object of static
```

Comment:
How would you feel about `\defnx{function-local predefined variable}{variable!function-local predefined}`?